American Whitewater/Helpers/AWApiReachHelper.swift

```diff
@@ -567,9 +565,11 @@ class AWApiReachHelper {
     
     
     func updateAllReachDistances(callback: @escaping UpdateCallback) {
+        let coord = DefaultsManager.shared.coordinate
         
-        if DefaultsManager.latitude == 0.0 || DefaultsManager.longitude == 0.0 {
-            print("Unable to update distance - user location is \(DefaultsManager.latitude)x\(DefaultsManager.longitude)")
+        // FIXME: probably CLLocationCoordinate2DIsValid() is what's wanted here?
+        if coord.latitude == 0.0 || coord.longitude == 0.0 {
```

Comment:
Is the coordinate guaranteed to exist?

Reply:
Currently, it is -- DefaultsManager.latitude/longitude defaulted to 0 if it hadn't been set. I didn't change that, so the coordinate will default to (0, 0).

Would probably be better to make those optional, and `if let` unwrap it in places like this where seems like maybe the question is, did we store a user location?

Reply:
I thought there was a case where the system defaulted to 0,0 (not us), although it might've been us all the time. It would probably be better to use an optional, but doesn't seem like a priority.